p9Layouts/defaultPnorLayout_64.xml

```diff
@@ -160,10 +160,20 @@ Layout Description
         <ecc/>
     </section>
     <section>
-        <description>Hostboot Data (1.125M)</description>
-        <eyeCatch>HBD</eyeCatch>
+        <description>Hostboot Data - READ-ONLY (~819KB)</description>
+        <eyeCatch>HBD_RO</eyeCatch>
         <physicalOffset>0x305000</physicalOffset>
-        <physicalRegionSize>0x120000</physicalRegionSize>
+        <physicalRegionSize>0xCD000</physicalRegionSize>
```

Comment:
for 1MB-128k I'm getting around 0xFC000 which seems significantly different than your calc here.